### ISSUE TITLE
Collection view adjust size buttons

### ISSUE THREAD

**Issue by mpadovani** [SwipeCellKit/SwipeCellKit#242]
Can I adjust the width of the button the same for everyone?

Can I adjust the size of the button to a bit smaller than the collection view and center them?

Thank you

![size 2](https://user-images.githubusercontent.com/3639990/44819552-9a453780-abc3-11e8-82af-d73b4c6a278c.jpg)
![size 1](https://user-images.githubusercontent.com/3639990/44819553-9a453780-abc3-11e8-9047-9f3df7da5f10.jpg)

**Comment by mpadovani** [comment]
Ajusting UIEdgeInsets

**Comment by kurabi** [comment]
I recommend you use circular button style as well.

**Comment by mpadovani** [comment]
I'm circled now, but still having a lot of problems to adjust correct size, because using a CollectionView with shadow is with a strange result, not to mention that I can not adjust the Insets of a cell in real time

**Comment by mpadovani** [comment]
My UICollectionViewFlowLayout this problem I user 23 spacing left and right

```
import UIKit

class ListTestCollectionViewLayout: UICollectionViewFlowLayout {
    override func prepare() {
        super.prepare()
        setupCollectionViewLayout(collectionView!)
    }
    
    func setupCollectionViewLayout(_ collectionView: UICollectionView) {
        let spacing: CGFloat = 23.0
        scrollDirection = .vertical
        minimumLineSpacing = 23.0
        minimumInteritemSpacing = 23.0
        sectionInset = UIEdgeInsets(top: spacing, left: spacing, bottom: spacing, right: spacing)
        itemSize = self.itemSize(for: collectionView)
    }
    
    func set(headerReferenceSize: CGSize) {
        self.headerReferenceSize = headerReferenceSize
    }
    
    func itemSize(for collectionView: UICollectionView) -> CGSize {
        let safeWidth: CGFloat
        if #available(iOS 11.0, *) {
            safeWidth = UIScreen.main.bounds.width - collectionView.safeAreaInsets.right - collectionView.safeAreaInsets.left
        } else {
            safeWidth = UIScreen.main.bounds.width
        }
        
        let itemWidth = safeWidth - sectionInset.left - sectionInset.right
        let itemHeight: CGFloat = 110.0
        return CGSize(width: itemWidth, height: itemHeight)
    }
}
```

**Comment by kurabi** [comment]
The circular button style per the example Mail App.
